packages/component/src/Dictation.js

```diff
@@ -29,7 +29,7 @@ class Dictation extends React.Component {
     props.stopDictate();
 
     if (transcript) {
-      props.setSendBox(transcript, 'speech');
+      props.setSendBox(transcript);
```

Comment:
`setSendBox` no longer accept `via` argument. It was not used before, so it get cleaned up.